.github/workflows/build-and-test.yml

```diff
@@ -0,0 +1,64 @@
+name: Build and test
+on: [push]
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version:
+          - 2.7
+          - 3.5
+          - 3.6
+          - 3.7
+          - 3.8
+          - 3.9
+          - pypy2
+          - pypy3
+    env:
+      # ASTTOKENS_SLOW_TESTS: 1
+      COVERALLS_PARALLEL: true
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+
+      - name: Install dependencies
+        run: |
+          pip install --upgrade setuptools>=44 wheel setuptools_scm[toml]>=3.4.3 pep517
+          pip install --upgrade coverage
+          # python3 -m pep517.build . # replace the following command with this one when you drop python 2
+          python ./setup.py bdist_wheel
+          export WHLNAME=./dist/asttokens-0.CI-py2.py3-none-any.whl
+          mv ./dist/*.whl $WHLNAME
+          pip install --upgrade --pre "$WHLNAME[test]"
```

Comment:
I have a similar packaging setup in `executing` and Python 2 support, and I just use `pip install .` instead of all this: https://github.com/alexmojaki/executing/blob/master/.github/workflows/test.yml